### DISCUSSION TITLE
antsApplyTransforms BOLD error

### DISCUSSION THREAD

**Discussion by arianayoum** [ANTsX/ANTs#1459] [Q&A]
I'm trying to apply the transforms from my T1->template to my BOLD image. 

My code: 
  antsApplyTransforms \
    -d 3 \
    -e 3 \
    -i /data/fmriprep_preprocessed/output/fmriprep/sub-006/func/sub-006_task-MID_space-T1w_desc-preproc_bold.nii.gz \
    -r /data/ANTs/braintemplate/anat/colintoOA_BrainExtractionBraindeformed.nii.gz \
    -o /data/fmriprep_preprocessed/output/fmriprep/sub-006/func/sub-006_task-MID_colintoOA_desc-preproc_bold.nii.gz \
    -n NearestNeighbor \
    -t /data/ANTs_registered/regular/sub-006/anat/sub-006_BrainExtractionBrainWarp.nii.gz \
    -t /data/ANTs_registered/regular/sub-006/anat/sub-006_BrainExtractionBrainAffine.txt \
    -z 1

However, it keeps giving me an error: 
WARNING: In /usr/include/ITK-4.10/itkTransformFactoryBase.h, line 81
TransformFactoryBase (0x557f4ed630a0): Refusing to register transform "MatrixOffsetTransformBase_double_3_3" again!

Does anyone know what's going on?

**Comment by cookpa** [comment]
The reference to ITK 4.10 suggests you have a very old version of ANTs, an update may fix the problem.
